packages/core/parcel/README.md

```diff
@@ -1286,38 +1341,73 @@ asset graph. They mostly call out to different compilers and preprocessors.
 import {Transform} from '@parcel/plugin';
```

Comment:
This part of the docs seem to match up with the PR I created, #3075. Maybe what I wrote is not necessary and we can close that PR.